### PR TITLE
Mobile: Use native dropdown component

### PR DESCRIPTION
**Pull request by cvarley100**
# Description

- Replace custom dropdown component with React Native picker

## Type of change

- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?

Tested on iOS and Android

# Checklist:

_Please delete items that are not relevant._

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] New and existing unit tests pass locally with my changes
- [ ] For changes to `mobile` that include native code (including React Native modules): I have verified that both iOS and Android successfully build in both `Debug` and `Release` modes

### CODE REVIEW

File: src/mobile/src/ui/views/onboarding/LanguageSetup.js [hunk 1]
```diff
@@ -115,7 +114,7 @@ class LanguageSetup extends Component {
         return (
             <TouchableWithoutFeedback
                 onPress={() => {
-                    if (this.dropdown) {
+                    if (isIOS) {
```
Comment (laumair) [inline src/mobile/src/ui/views/onboarding/LanguageSetup.js:117]: Why are we checking platform here? Do we not want to close the dropdown on android?

**Review by laumair — APPROVED** [review]
Looks good overall.

Slightly confused about https://github.com/iotaledger/trinity-wallet/pull/1275/files#r275277506

File: src/mobile/src/ui/views/onboarding/LanguageSetup.js (hunk 1 above)
Comment (cvarley100) [reply src/mobile/src/ui/views/onboarding/LanguageSetup.js:117]: Android dropdown covers the whole screen, so you can't click on the bg to close it

Comment (cvarley100) [reply src/mobile/src/ui/views/onboarding/LanguageSetup.js:117]: You need to test it to see what I mean

**Pull request merged**